client.go

```diff
@@ -48,54 +47,54 @@ var (
 )
 
 const (
-	// DefaultAddress is the default IPV4 address that is used for a connection
-	DefaultAddress = "127.0.0.1"
-	// DefaultPort is the default port used for a connection
-	DefaultPort     = 6640
-	UNIX            = "unix"
-	TCP             = "tcp"
-	SSL             = "ssl"
-	SKIP_TLS_VERIFY = true
+	DefaultTCPAddress  = "127.0.0.1:6640"
+	DefaultUnixAddress = "/var/run/openvswitch/ovnnb_db.sock"
```

Comment:
I don't think we need 2 default addresses. The default can be either TCP or Unix. We should follow C/Python ovsdb client patterns. There were no use case that user specifies protocol but omits the actual address.

Reply:
i create this variables for end-user, so if you can't connect you can see this variables and check own configuration of ovn.

Reply:
i'm un-export this variables..